lib/plugin/github.js

```diff
@@ -54,4 +82,23 @@ module.exports = function (md, opts) {
       return md.renderer.renderToken(tokens, idx, options)
     }
   }
+
+  // rewrite image locations in inline HTML to be fully qualified github URLs
+  var originalHtmlInlineRule = md.renderer.rules.html_inline
+  md.renderer.rules.html_inline = function (tokens, idx, options, env, self) {
+    var content = tokens[idx].content
+    var repoPath = path.join(repo.user, repo.repo, 'master')
+    var imageBaseUrl = 'https://raw.githubusercontent.com/' + repoPath
+    var newHtml
```

Comment:
I'm not super happy with the nested `if()` structure here, but just doing, e.g.,
```js
newHtml = replaceTagAttribute(content, 'img', 'src', imageBaseUrl + '/') ||
          replaceTagAttribute(content, 'a', 'href', repo.https_url + '/blob/master/')
```
looked bad when the `standard` formatting was applied (leading whitespace on the second line gets removed)